### DISCUSSION TITLE
Modifying macOS brightness keys to F1 and F2 only works when computer display is open

### DISCUSSION THREAD

**Discussion by akshathjain** [MonitorControl/MonitorControl#1455] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

I went into System Settings -> Keyboard -> Keyboard Shortcuts... -> Display -> Decrease / Increase Display brightness and changed those to F1 and F2, respectively (originally F14, F15). 

I'm only able to use F1 and F2 to change my monitor displays when my laptop screen is open. When it's closed, monitor control doesn't seem to work. 

### Steps to reproduce

1. Go to System Settings -> Keyboard -> Keyboard Shortcuts... -> Display -> Decrease / Increase Display brightness
2. Change to F1 and F2
3. Observe that monitor control works when the laptop display is open (but there's no brightness change indicator)
4. Close laptop display
5. Monitor control doesn't work

### Expected behavior

Display brightness should change when laptop screen is closed.

### Environment Information (please complete the following information)

```markdown
- macOS version: Ventura 
- Mac model: Macbook Pro
- MonitorControl version: Latest
- Monitor(s): Dell P2418D
- Apple Silicon/M1 (yes or no): yes
```

**Comment by waydabber** [comment]
You can set custom shortcuts in MonitorControl itself, that should work for all screens.
